### PR TITLE
BETA: Register alf.is-a.dev

### PR DESCRIPTION
**Pull request by alfonsop123**
Added `alf.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/alf.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "alfonsop123",
+        "email": "alfthemonster123@gmail.com"
+    },
+    "record": {
+        "CNAME": "gv-gybmprmm6adobi.dv.googlehosted.com"
```
Comment (DEV-DIBSTER) [inline domains/alf.json:7]: No site content. Unable to reach.

File: domains/asddsadasads.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "alfonsop123",
+        "email": "alfthemonster123@gmail.com"
+    },
+    "record": {
+        "TXT": "google-site-verification=wwoshgzb1e_med696nh0rsnw8pzzmbkmor-94rscdr0"
```
Comment (DEV-DIBSTER) [inline domains/asddsadasads.json:7]: TXT record alone doesn't do anything.

File: domains/test.json
Comment (DEV-DIBSTER) [inline domains/test.json:1]: Domain will not be released due to abuse risk.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/test.json
Comment (alfonsop123) [reply domains/test.json:1]: bro what this is for myself but it said i had to verify it on google sites and the dashboard never loaded the alf.is-a.dev

**Comment by alfonsop123** [comment]
Hello

**Pull request closed without merging**